cloudformation/find-workers.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -e
+
+if [ -z "$AWS_DEFAULT_REGION" ]; then
+    export AWS_DEFAULT_REGION=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/document | grep region | awk -F\" '{print $4}')
```

Comment:
@marcocitus would know best but I think the answer is no. The documentation suggests using the ip address http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html

With the level of control AWS has over the network, this ip is basically a dns name ;)